exasol_integration_test_docker_environment/lib/api/spawn_test_environment.py

```diff
@@ -0,0 +1,81 @@
+    if parsed_db_mem_size < humanfriendly.parse_size("1 GiB"):
+        raise ArgumentConstraintError("db_mem_size", "needs to be at least 1 GiB")
+    parsed_db_disk_size = humanfriendly.parse_size(db_disk_size)
+    if parsed_db_disk_size < humanfriendly.parse_size("100 MiB"):
+        raise ArgumentConstraintError("db_disk_size", "needs to be at least 100 MiB")
+    set_build_config(False,
+                     tuple(),
+                     False,
+                     False,
+                     output_directory,
+                     temporary_base_directory,
+                     None,
+                     None)
+    set_docker_repository_config(source_docker_password, source_docker_repository_name, source_docker_username,
+                                 source_docker_tag_prefix, "source")
+    set_docker_repository_config(target_docker_password, target_docker_repository_name, target_docker_username,
+                                 target_docker_tag_prefix, "target")
+    task_creator = lambda: generate_root_task(task_class=SpawnTestEnvironmentWithDockerDB,
+                                              environment_name=environment_name,
+                                              database_port_forward=str(
+                                                  database_port_forward) if database_port_forward is not None else None,
+                                              bucketfs_port_forward=str(
+                                                  bucketfs_port_forward) if bucketfs_port_forward is not None else None,
+                                              mem_size=db_mem_size,
+                                              disk_size=db_disk_size,
+                                              nameservers=nameserver,
+                                              docker_runtime=docker_runtime,
+                                              docker_db_image_version=docker_db_image_version,
+                                              docker_db_image_name=docker_db_image_name,
+                                              db_user="sys",
+                                              db_password="exasol",
+                                              bucketfs_write_password="write",
+                                              no_test_container_cleanup_after_success=True,
+                                              no_test_container_cleanup_after_failure=False,
+                                              no_database_cleanup_after_success=True,
+                                              no_database_cleanup_after_failure=False,
+                                              is_setup_database_activated=not deactivate_database_setup,
+                                              create_certificates=create_certificates
+                                              )
+    success, task = run_task(task_creator, workers, task_dependencies_dot_file)
+    return success
```

Comment:
Here would it cool, if we in future can return the EnvironmentInfo and cleanup method , but this is something for another PR

Reply:
That would be the next one on my list :)